### DISCUSSION TITLE
Integration with MacOS Shortcuts App

### DISCUSSION THREAD

**Discussion by fibonacid** [rxhanson/Rectangle#621] [Ideas]
It would be very nice to be able to create screen profiles through the Shortcuts app because what MacOS offers is quite limited.

![Schermata 2021-11-02 alle 09 39 10](https://user-images.githubusercontent.com/37226716/139813193-40fa7241-51ea-4d8c-8597-b1a76f76a1e6.jpg)

Is there a way to control Rectangle from the command line until the direct integration is in consideration or development?
.

**Comment by rxhanson** [comment]
I don't have the bandwidth for either Shortcuts integration or command line integration at the moment. PR's welcome!
